realm/realm-annotations-processor/src/main/java/io/realm/processor/ClassMetaData.java

```diff
@@ -332,11 +337,13 @@ private boolean checkDefaultConstructor() {
 
     private boolean checkForFinalFields() {
         for (VariableElement field : fields) {
-            if (field.getModifiers().contains(Modifier.FINAL)) {
-                Utils.error(String.format(Locale.US,
-                        "Class \"%s\" contains illegal final field \"%s\".", className, field.getSimpleName().toString()));
-                return false;
-            }
+            if (!field.getModifiers().contains(Modifier.FINAL)) { continue; }
```

Comment:
Sure.  I am the person that introduced the code formatter and I did run it on this code.

I believe that the team policy is that if the formatter accepts it, it is ok.  We can change the formatter, if you think it important.  The point, however, is that delaying a commit for yet another day, for an un-automated style complaint, is a significant impediment to cross-time-zone development.